index.html

```diff
@@ -39,21 +32,21 @@
         <button class="mobile-menu" type="button"><img src="https://img.icons8.com/ios/50/000000/menu--v4.png" alt="hamburger-menu"/></button>
         <div class="mobile-menu-list">
           <a class="nav-item, nav-link" href="about.html">About</a>
-          <a class="nav-item, nav-link" href="#mainProgram">Program</a>
+          <a class="nav-item, nav-link" href="#mainprogram">Program</a>
           <a class="nav-item, nav-link" href="#">Join</a>
           <a class="nav-item, nav-link" href="#sponsor">Sponsor</a>
           <a class="nav-item, nav-link" href="#">News</a>
           <a class="nav-item, nav-link" href="#">CC Compaign</a>
         </div>
         <h1 class="world">"Hello! Sharing world"</h1>
         <h2 class="title">MUSIC FESTIVAL<br>EXTRAVAGANZA<br>2022</h2>
```

Comment:
- [ ] There is a background color that is blocking your `h2` text from displaying. It would be great if you could check the styles so that this heading displays well.
- Your design:
![Capture5](https://user-images.githubusercontent.com/85113567/153715377-85c4462b-287c-4381-a15b-f14ecefc67d4.PNG)

Reply:
Have an issue with style.css line 94. background-clip: text. The linters sees it as an error, when I use -webkit-background-clip: text; which I got from stackoverflow it also show as an error.  How do I ignore one line of code from linter checking.

Reply:
It's better to find an alternative way of achieving what you want rather than disabling linters for one line of code.

Reply:
This issue has not been resolved